src/test/extension.test.ts

```diff
@@ -19,92 +104,114 @@ const data = [
-        },
-    },
-    {
-        range: new sourcegraph.Range(1, 0, 1, 0),
-        isWholeLine: true,
-        after: {
-            backgroundColor: '#f2736d',
-            color: 'rgba(255, 255, 255, 0.8)',
-            contentText: ' View logs in Sentry (❕)» ',
-            hoverMessage:
-                ' Please fill out the following configurations in your Sentry extension settings: repoMatch, fileMatch',
-            linkURL:
-                'https://sentry.io/organizations/sourcegraph/issues/?project=134412&query=is%3Aunresolved+cannot%20determine%20file%20path&statsPeriod=14d',
+        expected: {
+            range: new sourcegraph.Range(new sourcegraph.Position(1, 0), new sourcegraph.Position(1, 0)),
+            isWholeLine: true,
+            after: {
+                backgroundColor: '#f2736d',
+                color: 'rgba(255, 255, 255, 0.8)',
+                contentText: ' View logs in Sentry (❕)» ',
+                hoverMessage: ' Add Sentry projects to your Sentry extension settings for project matching.',
+                linkURL: 'https://sentry.io/organizations/sourcegraph/issues/',
+            },
         },
     },
     {
-        range: new sourcegraph.Range(1, 0, 1, 0),
-        isWholeLine: true,
-        after: {
-            backgroundColor: '#f2736d',
-            color: 'rgba(255, 255, 255, 0.8)',
-            contentText: ' View logs in Sentry (❕)» ',
-            hoverMessage: ' Add Sentry projects to your Sentry extension settings for project matching.',
-            linkURL: 'https://sentry.io/organizations/sourcegraph/issues/',
+        goal:
+            'match line based on common pattern, render warning link hinting to add projectId and render link to general issues page',
+        index: 1,
+        match: '',
+        missingConfigData: ['repoMatches', 'fileMatches'],
+        sentryProjectId: undefined,
+        expected: {
+            range: new sourcegraph.Range(new sourcegraph.Position(1, 0), new sourcegraph.Position(1, 0)),
+            isWholeLine: true,
+            after: {
+                backgroundColor: '#f2736d',
+                color: 'rgba(255, 255, 255, 0.8)',
+                contentText: ' View logs in Sentry (❕)» ',
+                hoverMessage: ' Add Sentry projects to your Sentry extension settings for project matching.',
+                linkURL: 'https://sentry.io/organizations/sourcegraph/issues/',
+            },
         },
     },
 ]
 
 describe('decorate line', () => {
-    for (const [i, deco] of data.entries()) {
+    beforeEach(async () => {
+        await sourcegraph.configuration.get().update('sentry.organization', 'sourcegraph')
+        await sourcegraph.configuration.get().update('sentry.projects', projects)
+    })
+
+    for (const deco of data) {
         it('decorates the line with the following goal: ' + deco.goal, () =>
```

Comment:
The "with the following" reads a bit clunky. Consider the wording from the other test `with the goal to` or anything more fluid

Reply:
Does it decorate the line or just return decorations? The current description implies a call to `setDecorations()`, but it seems to return the decorations